src/lib.rs

```
@@ -145,7 +149,7 @@ mod test {
             "a": "b",
             "c": "d"
         });
-        let result = transform(source, &spec);
+        let result = transform(source, &spec).unwrap();
```

Need to propagate the error

---

this is inside a test so seems like should be ok

---

Yes, indeed. I didn't notice that it in tests